app/workers/classifications_dump_worker.rb

```diff
@@ -1,7 +1,10 @@
 require 'csv'
 
 class ClassificationsDumpWorker
+  include Sidekiq::Worker
   include DumpWorker
+  include DumpMailerWorker
```

Comment:
normal dump workers (e.g. classification) sends emails, so ive extracted the dump email functionality to a separate module for use in the dump mailer worker.